src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side) {
+        this.side = side;
+    }
+
+    @Override
+    public Figure createRandomFigure() {
+        return new Square(getRandomInt());
+    }
```

Comment:
The `createRandomFigure` method should not be in the `Square` class. The `FigureSupplier` class is responsible for creating random figures. Also, don't use static methods or generate random values inside figure classes; let's generate random values in `FigureSupplier`.